tenseal/tensors/ckksvector.cpp

```diff
@@ -150,7 +159,7 @@ CKKSVector CKKSVector::add(CKKSVector to_add) {
 }
 
 CKKSVector& CKKSVector::add_inplace(CKKSVector to_add) {
-    if (this->context != to_add.context) {
+    if (!this->context->equals(to_add.context)) {
```

Comment:
I think we don't have to check if the two context are equals since SEAL operations (add, sub, mult, negate) will check for context and the parameters internally.

Reply:
Added https://github.com/OpenMined/TenSEAL/issues/112 for a better review